### DISCUSSION TITLE
Is scaling support, advised or not neccessary?

### DISCUSSION THREAD

**Discussion by nilslacroix** [shap/shap#2523] [Q&A]
So I want to use shap with a model. Naturally the model is tuned and was trained on a scaled dataset. This introduces problems when I visualize my results because they have gone through a RobustScaler for example and can not be interpreted directly anymore. So would you advise to backtransform all results or just use the unscaled features für xai/shap?

**Comment by nilslacroix** [comment]
I would also like to know if there is a paramter or function to change the ticks on the shap plots according to the scaler which was used in the pipeline to create the model. Or if I can define a custom scaler to do the backtransformation, use a custom functon to for example use a np.expm1 function on a log transformed target variable?

**Reply by amylee-lixinyi** [reply]
hi
you might find this useful?
https://shap.readthedocs.io/en/latest/example_notebooks/tabular_examples/linear_models/Explaining%20a%20model%20that%20uses%20standardized%20features.html
